src/harness/harness.ts

```diff
@@ -493,7 +493,7 @@ module Harness {
             export let readFile: typeof IO.readFile = ts.sys.readFile;
             export let writeFile: typeof IO.writeFile = ts.sys.writeFile;
             export let fileExists: typeof IO.fileExists = fs.existsSync;
-            export let log: typeof IO.log = console.log;
+            export let log: typeof IO.log = console.log.bind(console);

```

Comment:
Both would work. Technically `console.log` can accept more than one argument, but `IO.log` does not.